airbyte-ci/connectors/metadata_service/lib/tests/test_commands.py

```diff
@@ -36,12 +36,12 @@ def test_file_not_found_fails():
 
 # TEST UPLOAD COMMAND
 @pytest.mark.parametrize("uploaded", [True, False])
-def test_upload(mocker, valid_metadata_yaml_files, uploaded):
+def test_upload(mocker, valid_metadata_upload_files, uploaded):
     runner = CliRunner()
     mocker.patch.object(commands.click, "secho")
     mocker.patch.object(commands, "upload_metadata_to_gcs")
     commands.upload_metadata_to_gcs.return_value = uploaded, "blob_id"
-    metadata_file_path = valid_metadata_yaml_files[0]
+    metadata_file_path = valid_metadata_upload_files[0]
```

Comment:
Please check this - tests failed because one of the valid (according to validation) metadata files was invalid because it referenced docker images which didn't exist. 

Seems that the upload tests were testing fixtures from `validate` instead of `upload`, so I swapped them here.